pyodide_build/mkpkg.py

```diff
@@ -10,6 +10,12 @@
 PACKAGES_ROOT = Path(__file__).parent.parent / 'packages'
 
 
+def get_sdist_url_entry(json_content):
+    for entry in json_content['urls']:
+        if entry['filename'].endswith('.tar.gz'):
```

Comment:
It actually could be any of the formats supported by [shutil.unpack_archive](file:///usr/share/doc/python3-docs/html/library/shutil.html?highlight=unpack_archive#shutil.unpack_archive)...  but maybe this is the most common and good enough for now.

Reply:
OK, I force-pushed a new version that looks at all of the different extensions returned by `shutil.get_unpack_formats()`.